x-pack/plugins/transform/public/app/components/toast_notification_text.tsx

```diff
@@ -83,15 +85,23 @@ export const ToastNotificationText: FC<ToastNotificationTextProps> = ({
             </EuiButtonEmpty>
           </EuiModalFooter>
         </EuiModal>,
-        { theme$: theme.theme$ }
+        { theme, i18n: i18nStart }
       )
     );
   };
 
   return (
     <>
-      <pre>{previewText}</pre>
-      <EuiButtonEmpty onClick={openModal}>
+      {!inline && (
```

Comment:
```suggestion
      {!inline ? (
       <>
          <pre>{previewText}</pre>{' '}
        </>
      ) : null}
```

Reply:
Since `inline` is a `boolean`, the pattern with `&&` should be safe to use.